storage/pruning/pruningStorer.go

```diff
@@ -185,24 +183,27 @@ func initPruningStorer(
 	return pdb, nil
 }
 
-func initPersistersInEpoch(
+func (ps *PruningStorer) initPersistersInEpoch(
```

Comment:
:+1: 
in another PR, I think we can safely make `PruningStorer` unexported

Reply:
Added TODO.